.github/workflows/linux.yml

```diff
@@ -50,12 +49,11 @@ jobs:
       image: ${{ matrix.container }}
     env:
       MAKE_PARAMS: "-j 18"
-      LIBOQS_BRANCH: "0.10.1"
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Full build
-        run: OPENSSL_BRANCH=${{ matrix.ossl-branch }} ./scripts/fullbuild.sh
+        run: OPENSSL_BRANCH=${{ matrix.ossl-branch }} LIBOQS_BRANCH=0.10.1 ./scripts/fullbuild.sh
```

Comment:
This will fail if `${{ matrix.ossl-branch }}` is empty. Suggest changing the contents of the LIBOQS_BRANCH var above instead.